setup/dotfiles.sh

```diff
@@ -1,19 +1,70 @@
 #!/bin/sh
 source ~/dotfiles/setup/functions.sh
 
+
+dotfiles=(
+zsh
+bin
+aliases
+agignore
+gitconfig
+gitmessage
+gvimrc
+tmux.conf
+zshenv
+zshrc
+vimrc.bundles
+vimrc
+vim
+)
+
+nvimrcpath="~/.config/nvim/init.vim"
+nvimpath="~/.config/nvimrc"
+
+fancy_echo "Backup current config"
+today=`date +%Y%m%d`
+# for i in ; do [ -e $i ] && [ ! -L $i ] && mv $i $i.$today; done
```

Comment:
Are these commented out lines needed?

Reply:
can be removed. The two is my current using logic, the code underneath have special logic to handle neovim.